packages/builders/src/jest/jest.builder.spec.ts

```diff
@@ -4,11 +4,17 @@ jest.mock('jest');
 const { runCLI } = require('jest');
 import * as path from 'path';
 
+class TestJestBuilder extends JestBuilder {
```

Comment:
Is this so you can spy? Can you cast to `any` instead please?

Reply:
Done.